documents/src/pages/elements/interactive-chart.md

```diff
@@ -394,6 +394,14 @@ chart.config = {
     }
   ]
 };
+setTimeout(() => {
+  chart.chart.priceScale('').applyOptions({
```

Comment:
```suggestion
  chart.chart.priceScale().applyOptions({
```

Reply:
This will show an error It required empty string as an id.

Reply:
In my view, `ELF v7` must to follow the `TradingView Docs` by passing the `left` or `right` to the method to prevent the confused between `ELF v7 Docs` and `TrandingView Docs`

Reply:
You can't pass `left` or `right` to it. It suppose to be the `priceScaleId` and in this case it's equal ''.

Reply:
OK, I have read the Trading View Docs, they intend to use empty string for overlay feature (for me this API might be making developer confused)